### PR TITLE
add sphere and column hydrostatic balance tests

### PR DESCRIPTION
**Pull request by szy21**
This PR adds hydrostatic balance tests in sphere and column configurations. We should make sure these tests work when updating numerics.

### CODE REVIEW

**Comment by szy21** [comment]
bors try

File: examples/hybrid/define_post_processing.jl [hunk 1]
```diff
@@ -981,4 +991,17 @@ function custom_postprocessing(sol, output_dir)
...
+        plot(
+            vec(w_phy),
+            vec(Fields.coordinate_field(Y.f).z ./ 1000);
+            xlabel = "w [m/s]",
+            ylabel = "z [km]",
+            xlims = (-0.02, 0.02),
+            legend = false,
+        )
+    end
+    Plots.mp4(anim, joinpath(output_dir, "w.mp4"), fps = 10)
```
Comment (charleskawczynski) [inline examples/hybrid/define_post_processing.jl:1006]: Could we load the solution from the exported HDF5 files? I think post-processing with `sol` is destined to be deprecated

**Review by Zhengyu-Huang — APPROVED** [review]
Thanks for the great work! This PR looks good for me!

File: examples/hybrid/define_post_processing.jl (hunk 1 above)
Comment (szy21) [reply examples/hybrid/define_post_processing.jl:1006]: I didn't find a current post-processing example that uses the hdf5 files. Could you point me to an example? Maybe we can also work on it together if we want to change the post-processing now.

Comment (szy21) [reply examples/hybrid/define_post_processing.jl:1006]: If we do that I think it would be better to separate the change into a separate PR?

**Review by charleskawczynski — APPROVED** [review]

**Pull request merged**